### PR TITLE
simplify onbording

### PR DESCRIPTION
**Pull request by bjoern-michaelsen-freiheit**
- add a `make builder` target for easier initial setup
- make Dockerfiles not rely on proto being precompiled

### CODE REVIEW

File: README_DEVELOPERS.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The `cd-service` takes the URL of the repository to watch from the environment v
 
 You need a `builder` image that is tagged as `latest` to build services locally.
 There's 2 ways to get an image:
-* `IMAGE_TAG=latest make -C infrastructure/docker/builder build`
+* `make builder` (recommended for independant contributors)
```
Comment (sven-urbanski-freiheit-com) [inline README_DEVELOPERS.md:22]: I would argue this is recommended for everyone

File: services/cd-service/Dockerfile [hunk 2]
```diff
@@ -11,6 +11,7 @@ ADD services/cd-service/pkg /kp/services/cd-service/pkg
 ADD pkg /kp/pkg
 COPY go.sum go.mod /kp/
 
+RUN make -C /kp/pkg/api
```
Comment (sven-urbanski-freiheit-com) [inline services/cd-service/Dockerfile:14]: if the cd-service needs it, I would assume the rollout-service needs it too. -> please add it there.

**Review by sven-urbanski-freiheit-com — APPROVED** [review]

**Comment by bjoern-michaelsen-freiheit** [comment]
> I would argue this is recommended for everyone 

So should we then remove the other way altogether? Easier to maintain, what is gone can not bitrot.

File: services/cd-service/Dockerfile (hunk 2 above)
Comment (bjoern-michaelsen-freiheit) [reply services/cd-service/Dockerfile:14]: Done

File: README_DEVELOPERS.md (hunk 1 above)
Comment (bjoern-michaelsen-freiheit) [reply README_DEVELOPERS.md]: Simplified accordingly.

**Pull request merged**